Modules/Managers/PathManager.m

```diff
@@ -414,4 +317,83 @@ function remove_path_CB(obj,hObj,eventdata)
+                    % Even if false positive, worth checking
+                    temp_file = fullfile(fileparts(mfilename('fullpath')),'temp','transition.m');
+                    pre = ['% PathManger is trying to update your path instructions to the new,',...
+                           '% plain-code format. There was an issue updating "' path.name '".',...
+                           '% Edit the attempted update to be correct.' newline,...
+                           '%     You can use CommandCenter''s "**" menu to help with modules' newline];
+                    [instructions,~] = uigetcode(temp_file,path.name,pre,'',instructions,false);
+                end
+            end
+            path.alias = alias;
+            path.instructions = instructions;
+        end
+        function instruction_strings = get_string_instructions(instructions)
+            % Now exclusively used for backwards compatibility
+            instruction_strings = {};
+            if isstruct(instructions)
+                for i = 1:length(instructions)
+                    narin = length(instructions(i).method_inputs);
+                    method_inputs = cell(1,narin);
+                    for j = 1:narin
+                        method_inputs{j} = PathManager.format_instruction_inputs(instructions(i).method_inputs{j});
+                    end
+                    if ~isempty(instructions(i).module_name) % Module method
+                        narin = length(instructions(i).module_inputs);
+                        module_inputs = cell(1,narin);
+                        for j = 1:narin
+                            module_inputs{j} = PathManager.format_instruction_inputs(instructions(i).module_inputs{j});
+                        end
+                        % Instantiate into variable
+                        instruction_strings{end+1} = sprintf('a = %s.instance(%s)',...
+                            instructions(i).module_name,...
+                            strjoin(module_inputs,', '));
+                        % Then call method
+                        instruction_strings{end+1} = sprintf('a.%s(%s)',...
+                            instructions(i).method_name,...
+                            strjoin(method_inputs,', '));
+                    else % Function
+                        instruction_strings{end+1} = sprintf('%s(%s)',...
+                            instructions(i).method_name,...
+                            strjoin(method_inputs,', '));
+                    end
+                end
+            else
+                error('Instructions are in the wrong format.')
+            end
+        end
+        function fmt = format_instruction_inputs(input)
+            % Now exclusively used for backwards compatibility
+            if isnumeric(input)
+                fmt = ['[' num2str(input) ']'];
+            elseif ischar(input)
+                fmt = ['''' input ''''];
+            elseif iscell(input)
+                fmt = '??';
```

Comment:
This was changed from last time simply to be on the skeptical side when updating old prefs.